apps/src/block_utils.js

```diff
@@ -1121,6 +1123,19 @@ exports.createJsWrapperBlockCreator = function (
         }
         interpolateInputs(blockly, this, inputRows, inputTypes, inline);
         this.setInputsInline(inline);
+
+        // Use window.appOptions, not global appOptions, because the levelbuilder
+        // block page doesn't have appOptions, but we *do* want to show the mini-toolbox
+        // there
+        if (
+          miniToolboxBlocks &&
```

Comment:
You could use `showMiniToolbox` constant already declared and assigned above instead of condition/comment.

Reply:
I actually have this change made locally but had forgotten to push it! Thanks, Alice!